custom_components/tahoma/__init__.py

```diff
@@ -42,6 +44,15 @@
 SERVICE_EXECUTE_COMMAND = "execute_command"
 
 
+@dataclass
+class HomeAssistantOverkizData:
```

Comment:
Was a nice addition, avoid the magic string.

Reply:
yeah I like it, took it from Tuya integration